app/models/survey.rb

```diff
@@ -0,0 +1,5 @@
+class Survey < ActiveRecord::Base
```

Comment:
not really, I could just have the assessments evaluated directly from assessments. But then I would need an assessment controller and view?